### ISSUE TITLE
Issue with BPF file geometry transformation

### ISSUE THREAD

**Issue by msmitherdc** [PDAL/PDAL#3009]
[gridusr@mdsdev ~]$ docker exec -i pdaldock pdal info --metadata --boundary /test/pointcloud/fixtures/eo_pc_private.bpf
PDAL: Geometry::transform() failed.

```
      "compoundwkt": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]",
```

```
        "bbox":
        {
          "maxx": 467505.2458,
          "maxy": 218209.2666,
          "maxz": 1303.986114,
          "minx": 467382.2458,
          "miny": 218149.5166,
          "minz": 1284.213241
        },
```
bpf file: https://github.com/CRREL/GRiD/blob/releases/v1.4.8.2/apps/pointcloud/fixtures/eo_pc_private.bpf

This is new with 2.1, no issues with 2.0

**Comment by abellgithub** [comment]
Have you changed your PROJ version?  That seems the most likely culprit.

**Comment by msmitherdc** [comment]
This is a conda install, it is a new proj version, 6.3.1, as part of the 2.1 conda install

**Comment by abellgithub** [comment]
The issue here is that PDAL is catching an error in the transformation that was uncaught in the past.  I'm not sure, though, what to do.  I think we'd give bogus area and density numbers without this transformation.  I tend to think throwing an error and exiting is the wrong behavior.

**Comment by abellgithub** [comment]
Addressed in #3026
